### DISCUSSION TITLE
New comments in incident not showing (probably browser cache?)

### DISCUSSION THREAD

**Discussion by BBaoVanC** [upptime/upptime#260] [Q&A] [answered]
When I add new comments to an incident, they don't show up unless I load the page in a private window. The weird thing is that if I shift-click the reload button in Firefox, it still doesn't show the new comments (even though it should be loading without cache). Is this something weird in my browser settings?

**Comment by AnandChowdhary** [comment, marked as answer]
Yes, the GitHub API caches some data aggressively. There's not really much we can do, because the browser respects the cache-control headers. The shift-reload might also not work because GitHub also caches API responses on the server side, but new comments show up in a few minutes usually. :)
